t/plugin/limit-conn.t

```diff
@@ -1225,3 +1225,61 @@ property "burst" is required
 done
 --- no_error_log
 [error]
+
+
+
+=== TEST 32: enable plugin: conn=1
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                [[{
+                    "plugins": {
+                        "limit-conn": {
+                            "conn": 1,
+                            "burst": 0,
+                            "default_conn_delay": 0.3,
+                            "rejected_code": 503,
+                            "key": "remote_addr"
+                        }
+                    },
+                    "upstream": {
+                        "nodes": {
+                            "127.0.0.1:1980": 1
+                        },
+                        "type": "roundrobin"
+                    },
+                    "uri": "/hello"
+                }]]
+            )
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 33: hit route and should not be limited
+--- pipelined_requests eval
```

Comment:
Shouldn't the request be rejected here?  I did not understand

Reply:
All requests are sent one after another, they are not concurrent.

So the request should never be limited for `"conn": 1,`.

In the old code, it will be limited due to this bug.